bins/recipe-runner/src/runner.c

```diff
@@ -631,7 +638,12 @@ GglError runner(const RecipeRunnerArgs *args) {
         &resp
     );
     switch (ret) {
+    case GGL_ERR_NOMEM:
+        GGL_LOGW("Failed to get noProxyAddresses from config - lack of memory."
+        );
+        break;
```

Comment:
Same, this should exit.

Reply:
Also for both, rather than "lack of memory", "value longer than supported" would be more descriptive.

Reply:
ack

Reply:
done